examples/test_notebooks.py

```diff
@@ -0,0 +1,72 @@
+import pathlib
```

Comment:
Can we put a docstring at the top of this file explaining what the purpose of this is?